File: public/usage-examples/graphics/text_height_font_named-1-example.py
from splashkit import *

open_window("Text Height", 680, 200)
clear_screen(color_white())

load_font("LOTR", "lotr.TTF")
set_font_style_name_as_string("LOTR", font_style.bold_font)
draw_text_font_as_string("Ringbearer", color_black(), "LOTR", 100, 30, 20)

#Getting the height of the text to fill a rectange of that height
height = text_height_font_named("Ringbearer", "LOTR", 100)
fill_rectangle(color_black(), 20, 20, 10, height)
refresh_screen()

delay(5000)
close_all_windows()

File: public/usage-examples/graphics/text_height_font_named-1-example.cpp
#include "splashkit.h"

int main()
{
    open_window("Text Height", 680, 200);
    clear_screen(COLOR_WHITE);

    load_font("LOTR", "lotr.TTF");
    set_font_style("LOTR", BOLD_FONT);
    draw_text("Ringbearer", COLOR_BLACK, "LOTR", 100, 30, 20);
    
    int height = text_height("Ringbearer", "LOTR", 100);
    fill_rectangle(COLOR_BLACK, 20, 20, 10, height);
    refresh_screen();

    delay(5000);
    close_all_windows();

    return 0;
}